framework/include/cppmicroservices/detail/Log.h

```diff
@@ -64,13 +65,13 @@ class LogSink : public MultiThreaded<>, public std::enable_shared_from_this<LogS
 
 struct LogMsg {
 
-  LogMsg(LogSink& sink, const char* file, int ln, const char* func)
+  LogMsg(LogSink& sink, const std::thread::id tid, const char* file, int ln, const char* func)
     : enabled(false), buffer(), _sink(sink)
   {
     enabled = _sink.Enabled();
     if (enabled)
     {
-      buffer << "In " << func << " at " << file << ":" << ln << " : ";
+      buffer << "[" << tid << "] " << "In " << func << " at " << file << ":" << ln << " : ";
```

Comment:
Can't we just call `std::this_thread::get_id()` in-place here, instead of passing it as an additional argument to the `LogMsg` constructor? This would avoid the default argument and `thread` include directive in the header.